contracts/src/v0.8/automation/upkeeps/CCIPAutoWatchlist.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import "../../shared/access/ConfirmedOwner.sol";
+import {AutomationCompatibleInterface} from "../interfaces/AutomationCompatibleInterface.sol";
+import {AccessControl} from "../../vendor/openzeppelin-solidity/v4.8.3/contracts/access/AccessControl.sol";
+import {LinkAvailableBalanceMonitor} from "../upkeeps/LinkAvailableBalanceMonitor.sol";
+
+
+struct Log {
```

Comment:
can you import this and ILogAutomation from the existing `ILogAutomation.sol` interface?